### ISSUE TITLE
Github Action: no go files to analyze

### ISSUE THREAD

**Issue by hazcod** [golangci/golangci-lint#855]
Github action: https://github.com/ironPeakServices/iron-redis/blob/master/.github/workflows/pr.yaml#L4
```yaml
on: pull_request
name: PR Checks
jobs:
  golanglint:
    name: golang lint
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@master
    - uses: actions-contrib/golangci-lint@master
      with:
        args: run ./healthcheck/
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

However, it always complains about missing Go files, e..g https://github.com/ironPeakServices/iron-redis/commit/8e6df88762d8969a148ff6e7946c7cdb36c388db/checks?check_suite_id=310531360
```
msg="Running error: context loading failed: package github.com/go-redis/redis: no go files to analyze"
```

Any ideas? I have .Go files in `healthcheck/`.
The `github.com/` directory seems to be in `vendor/`, which should automatically be ignored since `--skip-dirs` is the default, right?

**Comment by cyw3** [comment]
the same error.

**Comment by hazcod** [comment]
Caused by https://github.com/golangci/golangci-lint/issues/865

**Comment by fdcds** [comment]
Probably a duplicate of #825.

**Comment by jirfag** [comment]
Duplicate of #825.
Also, I've made golangci/golangci-lint-action#15
